tidb-cloud/serverless-faqs.md

```diff
@@ -20,29 +20,56 @@ TiDB Serverless offers the TiDB database with full HTAP capabilities for you and
+
+Usage beyond the free quota will be charged. Once the free quota of a cluster is reached, the read and write operations on this cluster will be throttled until you [increase the quota](/tidb-cloud/manage-serverless-spend-limit.md#update-spend-limit) or the usage is reset upon the start of a new month.
 
 For more information, see [TiDB Serverless usage quota](/tidb-cloud/select-cluster-tier.md#usage-quota).
 
-### What does it mean for beta release?
+### How can I estimate the number of RUs required by my workloads and plan for monthly budget?
 
-TiDB Serverless is in beta while we continuously add new features and improve existing features before it becomes generally available. We do not provide SLA for beta products. Therefore, TiDB Serverless should **NOT** be used in production currently.
+To estimate the RU consumption of individual SQL statements, you can use the [EXPLAIN ANALYZE](/sql-statements/sql-statement-explain-analyze.md) SQL statement. You can see the RUs and storage your cluster has used in the Usage this month section of the Cluster Overview page. With past resource usage data and real-time resource usage graphs available on the Usage this month page, you can establish a reasonable spend limit for your cluster with the Edit Spend Limit feature. See [Manage Spend Limit for TiDB Serverless clusters](/tidb-cloud/manage-serverless-spend-limit.md) for more information.
 
-### What are the limitations of a TiDB Serverless cluster in beta?
+### How the storage is being metered for TiDB Serverless？
 
-For each organization in TiDB Cloud, you can create a maximum of five TiDB Serverless clusters by default. To create more TiDB Serverless clusters, you need to add a credit card and set a [spend limit](/tidb-cloud/tidb-cloud-glossary.md#spend-limit) for the usage.
+The storage is metered based on the amount of data stored in the TiDB cluster, measured in GB-months. This calculation is based on the sum of the size of all the tables and indexes in the cluster, without including data compression or replicas for high availability, and is multiplied by the number of hours the data is stored in the month.
 
-Some of TiDB Cloud features are partially supported or not supported on TiDB Serverless. See [TiDB Serverless Limitations and Quotas](/tidb-cloud/serverless-limitations.md) for details.
+### Why does the storage used size remain unchanged after dropping a table or database immediately?
 
-### What can TiDB Serverless be used for?
+TiDB retains dropped tables and databases for a period of time to ensure that transactions that depend on these tables can continue to run smoothly. Moreover, the extended retention time enables TiDB to provide the [FLASHBACK TABLE](/sql-statements/sql-statement-flashback-table.md)/[FLASHBACK DATABASE](/sql-statements/sql-statement-flashback-database.md) feature, which allows you to recover dropped tables and databases in the event that they were mistakenly deleted.
```

Comment:
```suggestion
This is because TiDB retains dropped tables and databases for a certain period of time. This retention period ensures that transactions dependent on these tables can continue execution without disruption. Additionally, the retention period makes the [FLASHBACK TABLE](/sql-statements/sql-statement-flashback-table.md)/[FLASHBACK DATABASE](/sql-statements/sql-statement-flashback-database.md) feature feasible, which allows you to recover dropped tables and databases if they were mistakenly deleted.
```